content/kubeone/master/guides/addons/_index.en.md

```diff
@@ -81,9 +85,105 @@ the path is relative to the KubeOne configuration file. This means that
 
 {{% notice note %}}
 Subdirectories are not considered when applying addons. Only addons in the root
-of the provided directory will be applied.
+of the provided directory will be applied. E.g. `./addons/addon_name1` is
+supported addon while `./addons/groupping/addon_name1` and
+`./addons/addon_name1/subdir1` does not.
+{{% /notice %}}
+
+## Embedded Addons
+
+Since v1.3 release KubeOne has new kind of addons that always caried with the
+binariy itself, embedded addons. This is done thanks to new Go's
+[`embed`][embed-docs] package. 
+
+Some of those addons are directly and automatically used by the KubeOne itself
+(like machinecontroller for example). But it's not the case for all of them.
+Some very useful addons need to be activated by the user in order to be deployed
+to the cluster, like `backups-restic`, `cluster-autoscaler` or
+`unattended-upgrades`.
+
+### Activate Embedded Eddons
+
+Or order to activate such embedded addons, user need to use new Addons
+[API][addons-api].
+
+Example:
```

Comment:
```suggestion
Example:

```

Add a new line to make markdown linters happy. :slightly_smiling_face:

Reply:
👍